code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -59,6 +59,17 @@
 
 	switch(M.a_intent)
 		if(I_HELP)
+			if(can_operate(src))
+				if(health >= config.health_threshold_crit)
+					if(src.surgeries.len)
+						for(var/datum/surgery/S in src.surgeries)
+							if(istype(S.get_surgery_step(), /datum/surgery_step/cavity/place_item) || istype(S.get_surgery_step(),/datum/surgery_step/remove_object))

```

Comment:
why do you check for surgeries by type here, just see if the next_step returns 1, if it doesn't, let the proc continue